client/app/scripts/superdesk-search/search.js

```diff
@@ -1202,8 +1202,8 @@
             };
         })
 
-        .directive('sdMultiActionBar', ['asset', 'multi', 'multiEdit',
-        function(asset, multi, multiEdit) {
+        .directive('sdMultiActionBar', ['asset', 'multi', 'multiEdit', 'packages', 'superdesk', 'notify',
+        function(asset, multi, multiEdit, packages, superdesk, notify) {
             return {
                 controller: 'MultiActionBar',
                 controllerAs: 'action',
```

Comment:
I noticed that some of the methods and values are bound to the scope, while the others are bound to the controller object - I suggest we make this consistent.

(preferably bind everything to controller, it's a more modern pattern in Angular).